src/physlr-indexlr.cc

```diff
@@ -146,6 +192,9 @@ main(int argc, char* argv[])
 	} else if (infiles.empty()) {
 		printErrorMsg(progname, "missing file operand");
 		failed = true;
+	} else if (withRepeat && withSolid) {
```

Comment:
Can you clarify what type of error this statement is checking for?

Reply:
That was something I left in when I first wrote the code. Wasn't sure how to approach using both Bloom filter. I will remove this line